src/scripts/plugin.js

```diff
@@ -85,7 +84,14 @@ const dialogConfig = (editor) => ({
     const dialog = this;
     const language = getLanguage(editor);
     dialog.setValueOf('tab-basic', 'language', language);
-    activateThebelab(thebelabConfig);
+  },
+  onCancel() {
+    const dialog = this;
+    const editorLanguage = getLanguage(editor);
+    const dialogLanguage = dialog.getValueOf('tab-basic', 'language');
+    if (editorLanguage !== dialogLanguage) {
+      activateThebelab(thebelabConfig);
```

Comment:
@celine168 Why would you want to reactivate thebelab when the current language and the selected language is different? There will not be any thebelab instance on the page to view.

Reply:
My thought was that if you change the language in the dialog, but click `cancel`, then the current thebelab instance would contain the changed language, which is not the language used in the editor (even if you canceled). 

Since we reactivate thebelab everytime we open the dialog, I don't think this makes a difference. But, if we find a way so that we don't have to do this (and this would be way more efficient too if we don't have to reactivate it each time), then this may be needed.

Reply:
Ok, let's leave this in.